go.mod

```diff
@@ -63,6 +63,12 @@ require (
 	sigs.k8s.io/yaml v1.3.0
 )
 
+// the below fixes the "go list -m all" execution
+replace (
+	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.26.1
+	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.26.1
+)
```

Comment:
There is already a `replace` below, you need to merge together:

![image](https://github.com/user-attachments/assets/dbf0536e-2ed2-46a1-933e-1fdafe352bf0)

Reply:
sorry for my mistake. done